content/en/docs/2024.5/Guides/upgrade-cortex/2024.1-to-2024.3/single-server-without-ha/upgrade-web-application-server.md

```diff
@@ -11,15 +11,15 @@ This guide describes how to upgrade {{% ctx %}} Gateway from 2024.1 to 2024.3. P
 
 ## Make Installation Artefacts Available
 
-{{< section "/install-web-application-server/make-installation-artefacts-available.md">}}
+{{< section "/upgrade/2024.3/upgrade-web-application-server/make-installation-artefacts-available.md">}}
```

Comment:
I forgot this from last docs this should be:
1. Copy the following artefacts to a folder on the machine:

   * Cortex Innovation {{< version >}} - Gateway.zip
   * Cortex Innovation {{< version >}} - Web App Server Install Scripts.zip

1. Extract the `Cortex Innovation {{< version >}} - Web App Server Install Scripts.zip` zip file to a folder with the same name.